apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue

```diff
@@ -28,9 +30,11 @@
                         {{ $t('DASHBOARDS.WIDGET.TOTAL_FAILURE_AND_SEVERITY.FAILURE_RATE') }}
                     </p>
                     <p class="value">
-                        {{ state.failureRate }}%
+                        {{ state.failureRate ? state.failureRate : '--' }}%
```

Comment:
issue: Even the number 0 is false when automatically converted to boolean in JavaScript, so even if the value is 0 (not empty), it will be shown as ‘--’!